src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -463,19 +519,35 @@ public CourseDetailsBundle getCourseSummary(String courseId) throws EntityDoesNo
         return getCourseSummary(cd);
     }
     
+    /**
+     * @param instructorAttributes
+     * @return the {@link CourseSummaryBundle course summary}, including
+     *         its feedback sessions using the given {@link InstructorAttributes}.
+     * @throws EntityDoesNotExistException
+     */
     public CourseSummaryBundle getCourseSummaryWithFeedbackSessionsForInstructor(
-            InstructorAttributes instructor) throws EntityDoesNotExistException {
-        CourseSummaryBundle courseSummary = getCourseSummaryWithoutStats(instructor.courseId);
-        courseSummary.feedbackSessions.addAll(feedbackSessionsLogic.getFeedbackSessionListForInstructor(instructor));
+            InstructorAttributes instructorAttributes) throws EntityDoesNotExistException {
```

Comment:
In most of the files, EntityAttributes variables are simply named as the entity (without "Attributes"),  so changing it here would make it somewhat inconsistent with the rest of the files